src/plugins/event-plugins/PageViewPlugin.ts

```diff
@@ -1,17 +1,30 @@
-import { PAGE_ID_FORMAT } from '../../orchestration/Orchestration';
+import {
+    PAGE_ID_FORMAT,
+    PAGE_INVOKE_TYPE
+} from '../../orchestration/Orchestration';
 import { MonkeyPatch, MonkeyPatched } from '../MonkeyPatched';
 import { Plugin, PluginContext } from '../Plugin';
 
 export const PAGE_EVENT_PLUGIN_ID = 'com.amazonaws.rum.page-view';
 
 export type Push = (data: any, title: string, url?: string | null) => void;
 export type Replace = (data: any, title: string, url?: string | null) => void;
+export type ClickInfo = {
+    timestamp: number;
+};
 
 /**
  * A plugin which records page view transitions.
  *
  * When a session is initialized, the PageManager records the landing page. When
  * subsequent pages are viewed, this plugin updates the page.
+ *
+ * When AutoPageView is enabled, this plugin does the following:
+ * (1) Records initial page as INITIAL_LOAD during load()
+ * (2) Any subsequent route changes via Browser History API is intercepted to:
+ *      - Create a new Page object with pageInvokeType as ROUTE_CHANGE
+ *
+ * In order for accurate tracking in SPA, recommend using PATH_AND_HASH for pageId.
```

Comment:
question: What is important to communicate in this user documentation? 

> In order for accurate tracking in SPA, recommend using PATH_AND_HASH for pageId.

Why is PATH_AND_HASH needed for accurate tracking? What happens if it is not used? We probably don't want to recommend a format here, since the correct format depends on the application.